packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -75,6 +79,26 @@ function itemFilter<T extends string | number>(inputValue: string) {
 const noop = () => {};
 const asyncNoop = () => Promise.resolve([]);
 
+function logOptions(
+  items: Array<ComboboxOption<string | number>>,
```

Comment:
I think just passing the count here should be enough.  You might want to put this in a separate file too, as we may use this for `MultiCombobox`.

Reply:
Nice! I was thinking of something like this, using it in other components. If we use this for MultiCombobox, would it be possible to consider using it in the `Select` component as well?

Reply:
I think that should be fine, as long as we make sure we don't end up drowning in logs 😅

Reply:
I agree, we should make sure we are not logging excessively. With the RECOMMENDED_ITEMS_AMOUNT though, we should hopefully only be getting the log if someone is using a component to display an abnormally large amount of options. For example, Prometheus 🥲, which has some customers and consumers who push the boundaries for options. Once we have this in production, I will be able to look for the logs in Faro. Maybe we won't even need the warning if Faro is alerting teams that consume excessive amounts of options?